ERCS/erc-741.md

```diff
@@ -0,0 +1,157 @@
+---
+eip: 741
```

Comment:
```suggestion
eip: 7647
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename and any references to use the issued number.